run.py

```diff
@@ -0,0 +1,141 @@
+import argparse
+import json
+import imp
+from termcolor import colored
+from configs.config import *
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--browser", help="(String) Browser tests are run on. "
+                                      "Options include: firefox, "
+                                      "firefox-headless, chrome, "
+                                      "chrome-headless. Default is "
+                                      "chrome-headless.",
+                    default=DEFAULT_BROWSER)
+
+
+def import_test(test_path, test_name):
+    """
+    Imports all test classes. Methods in classes are the tests, therefore,
+    we must dynamically import classes containing test methods in order to
+    execute tests.
+
+    :param test_path: (array) ordered list of packages in path to the test file
+    :param test_name: (string) name of the file w/ test, not including extension
+    :return:
+    """
+    # Define the test class name based on the test file name
+    # e.g., a test file named test_example has a class named TestExample
+    class_name = ""
+    for name in test_name.split("_"):
+        class_name = class_name + name.capitalize()
+
+    # Build the full file path to the test as a string
+    file_path = generate_file_path(test_path, test_name + ".py")
+
+    # Load module with test class given the file (test) name and file path
+    module = imp.load_source(test_name, file_path)
+
+    # Create instance of test class
+    test_class = getattr(module, class_name)(browser)
+
+    # Add instance of test class to dictionary
+    tests_loaded[test_name] = test_class
+
+
+def open_file(file_name):
+    """
+    Opens a file in read only mode and returns its content in JSON.
+
+    :param file_name: (strong) The name of the file to be opened
+    :return: (json) Contents of file
+    """
+    try:
+        with open(file_name, 'r') as f:
+            return json.load(f)
+    except Exception as e:
+        print "ERROR: Check your JSON file for syntax errors"
+        print e
+
+
+def generate_file_path(package_path, file_name):
+    """
+    Dynamically generate full path to file, including filename and extension.
+
+    :param package_path: (array) ordered list of packages in path to test file
+    :param file_name: (string) name of the file w/ test, including the extension
+    :return: (string) full path to file, including filename and extension
+    """
+    file_path = ""
+    for package in package_path:
+        file_path = file_path + package + "/"
+    return file_path + file_name
+
+
+def execute_tests(tests):
+    """
+    Executes specified tests.
+
+    :param tests: (json) List of tests to run
+    :return:
+    """
+    for test in tests:
+        # If a test has an enabled field set to true OR does not have an
+        # enabled field, then we assume the test is enabled and run it
+        if test.get("enabled", True):
```

Comment:
I think the best way to organize this logic is as follows:
```
if test ISN'T enabled
  continue

# IMPLIED ELSE
perform the test
```
This will save you some needless indentation, which may help make the code easier to read

Reply:
Good thought. Done.